client/my-sites/guided-transfer/transfer-unavailable-card.jsx

```diff
@@ -67,8 +64,7 @@ class TransferUnavailableCard extends Component {
 			<div>
 				<p>
 					{ translate(
-						`Howdy! It looks like there's something stopping us from being able
-				to transfer your site. Please {{a}}contact support{{/a}} and we'll sort it out!`,
+						"Howdy! It looks like there's something stopping us from being able to transfer your site. Please {{a}}contact support{{/a}} and we'll sort it out!",
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **5** times:
`translate( 'Howdy! It looks like there's something stopping us from being able
				to transfer your site. Please {{a}}contact support{{/a}} and we'll sort it out!' )` *ES Score: 9*